core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/DASH.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2020 Eclipse RDF4J contributors, and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
```

Comment:
This copyright header looks a bit odd: it's missing the full lines of asterisks at beginning and end.